### ISSUE TITLE
Reach out to AF team to kickstart migration

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#4446]
**What we're after:**
Reach out to AF to re-establish communication regarding the AF datatables "migration."

**Completion criteria:**
- [ ] Based on conversations with ADR/AF, we need to create followup tickets based on next steps

**Comment by JonellaCulmer** [comment]
We've reached out to AF and waiting on next steps.

**Comment by JonellaCulmer** [comment]
We have feedback and suggested changes from AF. Need to now determine whether those changes can be accommodated based on the underlying data and how it's structured. Some questions we still need to answer, technically:

- Can we move the initial fine and challenge received date from the summary to the disposition? Is this technically possible? Does that change the meaning for the Disposition section?
- Can we accurately and consistently replace the acronym RTB with Reason to believe?
- Can we accurately and consistently replace the penalty paid flag of Yes/No/Pending with paid/not yet paid/partially paid?
- Can we include the referral to Treasury line in the disposition accurately and consistently? Need to understand how it's in the API.

**Comment by JonellaCulmer** [comment]
Closing in favor of two new tickets:
- https://github.com/fecgov/openFEC/issues/4832 [Research] Add new endpoints to /legal/ for AFs
- https://github.com/fecgov/fec-cms/issues/4552 [Research] Front-end solutions for AF pages and document next steps
